modules/kubernetes-addons/karpenter/locals.tf

```diff
@@ -26,7 +26,7 @@ locals {
               clusterName: ${var.addon_context.eks_cluster_id}
               clusterEndpoint: ${var.addon_context.aws_eks_cluster_endpoint}
               defaultInstanceProfile: ${var.node_iam_instance_profile}
-              interruptionQueueName: ${var.sqs_queue_arn}
+              interruptionQueueName: ${var.sqs_queue_name}
```

Comment:
```suggestion
              interruptionQueueName: ${data.aws_arn.queue.resource}
```

Reply:
@bryantbiggs totally missed it `aws_arn` better choice, reverted to `sqs_queue_arn` since `aws_arn` providing resource name